src/elements/context-menu/context-menu.svelte

```diff
@@ -0,0 +1,11 @@
+<svelte:options immutable tag="v-context-menu" />
+
+<script lang="ts">
+import { addStyles } from '../../lib/index';
+
+addStyles();
+</script>
+
+<div class="py-1 border border-medium shadow-sm max-w-xs">
```

Comment:
Should this have an `aria-role` of [`menu`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role)?